src/sklearn_knn/_raw.py

```diff
@@ -1,13 +1,5 @@
-from sklearn.pipeline import Pipeline
+from ._base import IDNeighborsClassifier
 
-from ._base import IDNeighborsClassifier, KnnPipeline
 
-
-class Raw(KnnPipeline):
-    def _get_pipeline(self):
-        classifier = IDNeighborsClassifier(
-            n_neighbors=self.n_neighbors,
-            weights=self.weights,
-        )
-        steps = [("classifier", classifier)]
-        return Pipeline(steps)
+class Raw(IDNeighborsClassifier):
```

Comment:
Agreed on both counts! It's a little weird having an empty class, but I think it's preferable to the other options, and gives us some flexibility if we need to expand it later.

Now that you mention it, more explicit names for the estimators is probably a good idea. Not something we need to finalize right away, but if these are being imported alongside a bunch of other estimators, it would be nice to make it clear that they're classifiers.

Reply:
Yes, let's continue to track naming in a separate PR (class names as well as package name as a whole).  I've opened #6 to track this and note some things to think about.